src/problem_bank_scripts/problem_bank_scripts.py

```diff
@@ -1018,17 +1054,56 @@ def str_presenter(dumper, data2):
             encoding="utf8",
         )
 
-    # Move image assets
+    # Create the file errors list
+    os_errors = []
+
+    # Move client assets (generally images)
     files_to_copy = header.get("assets")
     if files_to_copy:
-        [copy2(pathlib.Path(source_filepath).parent / fl, output_path.parent) for fl in files_to_copy]
+        pl_path = output_path.parent
+        for file in files_to_copy:
+            try:
+                copy2(pathlib.Path(source_filepath).parent / file, pl_path / file)
+            except (FileExistsError, FileNotFoundError, IsADirectoryError, PermissionError) as e:
+                os_errors.append(str(e))
+
+    # Move server assets
+    files_to_copy = header.get("serverFiles")
+    if files_to_copy and instructor:
+        pl_path = output_path.parent
+        for file in files_to_copy:
+            try:
+                copy2(pathlib.Path(source_filepath).parent / file, pl_path / file)
+            except (FileExistsError, FileNotFoundError, IsADirectoryError, PermissionError) as e:
+                os_errors.append(str(e))
 
     # Move autograde py test files
     files_to_copy = header.get("autogradeTestFiles")
     if files_to_copy:
         pl_path = output_path.parent / "tests"
         pl_path.mkdir(parents=True, exist_ok=True)
-        [copy2(pathlib.Path(source_filepath).parent / "tests" / fl, pl_path / fl) for fl in files_to_copy if (instructor or fl=="starter_code.py")]
+        for file in files_to_copy:
+            if file != "starter_code.py" and not instructor:
+                continue
+            try:
+                copy2(pathlib.Path(source_filepath).parent / "tests" / file, pl_path / file)
+            except (FileExistsError, FileNotFoundError, IsADirectoryError, PermissionError) as e:
+                os_errors.append(str(e))
+    
+    # Move workspace files
+    files_to_copy = header.get("workspaceFiles")
+    if files_to_copy:
+        pl_path = output_path.parent / "workspace"
+        pl_path.mkdir(parents=True, exist_ok=True)
+        for file in files_to_copy:
+            try:
+                copy2(pathlib.Path(source_filepath).parent / "workspace" / file, pl_path / file)
+            except (FileExistsError, FileNotFoundError, IsADirectoryError, PermissionError) as e:
+                os_errors.append(str(e))
+
+    if os_errors:
+        error_msg = "\n    ".join(os_errors)
+        raise FileNotFoundError(f"Error(s) copying specified files:\n    {error_msg}")
```

Comment:
The idea behind the try-except system here is to show all file copy errors to the best of our ability at once, instead of having it show up one file at a time.

The output of this should look something as follows, as an example:
```python
FileNotFoundError: Error(s) copying specified files:
    [Errno 2] No such file or directory: 'test0'
    [Errno 2] No such file or directory: 'test1'
    [Errno 2] No such file or directory: 'test2'
    [Errno 2] No such file or directory: 'test3'
    [Errno 2] No such file or directory: 'test4'
    [Errno 2] No such file or directory: 'test5'
    [Errno 2] No such file or directory: 'test6'
    [Errno 2] No such file or directory: 'test7'
    [Errno 2] No such file or directory: 'test8'
    [Errno 2] No such file or directory: 'test9'
```